adafruit_is31fl3731.py

```diff
@@ -67,24 +108,28 @@ def _i2c_write_reg(self, reg, data):
             self.i2c.writeto(self.address, buf)
         finally:
             self.i2c.unlock()
+        return None
```

Comment:
This one is not needed, since there is no return in that function.

Reply:
yes, oops